src/App.tsx

```diff
@@ -10,18 +10,39 @@ import Navigation from 'components/Navigation';
 import Sidebar from 'components/Sidebar';
 import Wrapper from 'components/Wrapper';
 
+// Context
```

Comment:
Let's remove these comments :)

Reply:
Fixed in 4bf8642d6e8b1d3eab7e08c97ee4512e7d289920